### ISSUE TITLE
Inconsistent pipeline performance when disk space is exceeded

### ISSUE THREAD

**Issue by rcorces** [databio/pepatac#30]
This seems like it could be an important issue.
TL;DR - I ran the pipeline twice on certain samples and got vastly different file sizes for the aligned bams (differences of 2 - 3 GB) despite both giving a completed flag. I think this happened because I ran out of disk space.

I was looking at the total aligned reads for certain samples and some were alarmingly low (on the scale of 10,000 aligned reads) but the sample had an ATACseq_completed.flag.  I re-ran the pipeline on the same fastqs and got a very different result the second time. This is not an issue of non-deterministic vs deterministic. This looks like something failed the first time but wasnt caught and the pipeline forged onwards to completion.

My best guess is that this all happened during a single looper run where I ran out of disk space. 

I'm attaching two example logs (from the good run I just did and the truncated run that happened previously). You can see the spots where I ran out of disk space.

[Good_STAD-91C7155F-8187-4688-BEAB-C4179C786667-X029-S08-L015-B1-T1-P081_ATACseq_log.md.txt](https://github.com/databio/ATACseq/files/1536062/Good_STAD-91C7155F-8187-4688-BEAB-C4179C786667-X029-S08-L015-B1-T1-P081_ATACseq_log.md.txt)
[Good_THCA-ABF4154C-FACF-4711-AAC7-35C560F39BC2-X024-S10-L022-B1-T2-P064_ATACseq_log.md.txt](https://github.com/databio/ATACseq/files/1536063/Good_THCA-ABF4154C-FACF-4711-AAC7-35C560F39BC2-X024-S10-L022-B1-T2-P064_ATACseq_log.md.txt)
[Truncated_STAD-91C7155F-8187-4688-BEAB-C4179C786667-X029-S08-L015-B1-T1-P081_ATACseq_log.md.txt](https://github.com/databio/ATACseq/files/1536064/Truncated_STAD-91C7155F-8187-4688-BEAB-C4179C786667-X029-S08-L015-B1-T1-P081_ATACseq_log.md.txt)
[Truncated_THCA-ABF4154C-FACF-4711-AAC7-35C560F39BC2-X024-S10-L022-B1-T2-P064_ATACseq_log.md.txt](https://github.com/databio/ATACseq/files/1536065/Truncated_THCA-ABF4154C-FACF-4711-AAC7-35C560F39BC2-X024-S10-L022-B1-T2-P064_ATACseq_log.md.txt)
